src/color/creating_reading.js

```diff
@@ -1007,6 +1007,14 @@ p5.prototype.hue = function(c) {
  */
 p5.prototype.lerpColor = function(c1, c2, amt) {
   p5._validateParameters('lerpColor', arguments);
+
+  if (Array.isArray(c1)) {
```

Comment:
oh actually sorry, just noticed, we only convert to a color if it's an array, but I guess this now means we'd accept strings. Maybe these checks should be `!(c1 instanceof p5.Color)`?

Reply:
I've converted the checks to `!(c<n> instanceof p5.Color)` to support all color convertible types.